extensions/amp-horizontal-scroller/0.1/amp-horizontal-scroller.css

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software'
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+amp-horizontal-scroller {
+}
+.-amp-horizontal-scroller {
+  overflow-x: auto !important;
```

Comment:
Sure, an extension seems very much overkill but I wasn't clear on what was acceptable. I hadn't thought about the scheduling issue — in my case it would never occur (they're data tables).

Do you have an ETA for a solution? The problem we have is that this limitation essentially makes AMP unusable for the type of content we have. If it's likely to take a while I could live with a solution in which the descendants of a scrollable container are restricted to those that don't need to know where they are (which could largely boil down to a batch of `disallowed_ancestor` additions in validation).

Thanks for your feedback!